### PR TITLE
update js-soroban-client

### PR DESCRIPTION
**Pull request by tsachiherman**
update js-soroban-client to v0.3.0

### CODE REVIEW

File: packages/contracts/yarn.lock [hunk 1]
```diff
@@ -190,6 +190,21 @@ soroban-client@^0.2.1:
     stellar-base "8.0.1-soroban.6"
     urijs "^1.19.1"
```
Comment (Shaptic) [inline packages/contracts/yarn.lock:191]: Can this get dropped? The old version shouldn't be around at all.

Comment (tsachiherman) [reply packages/contracts/yarn.lock:191]: what are you referring to ? `8.0.1-soroban.6` ? afaict, this is the latest, isn't it ?

Comment (tsachiherman) [reply packages/contracts/yarn.lock:191]: its autogen via yarn.

**Comment by esteblock** [comment]
Hi, thanks!
What this version offer?
I was thinking in always follow https://soroban.stellar.org/docs/releases

What do you think?

**Comment by tsachiherman** [comment]
> 

It's the upcoming release, which will be officially release on Monday. I'd like to update it here, since the dapp example is using it - and I'd like to have the dapp example ready by Monday as well.

**Comment by tsachiherman** [comment]
> 

As for the content, this version add the getEvents() function

**Comment by esteblock** [comment]
Hello. You mean the https://github.com/stellar/soroban-example-dapp/ ?
It does still use soroban-client (js-soroban-client) version 0.2.1

**Comment by esteblock** [comment]
I went through https://github.com/stellar/js-soroban-client/commit/e0ad237d738be32b646f77ce69e9de2327382ab5
and it should work ok :)

**Comment by tsachiherman** [comment]
> I went through [stellar/js-soroban-client@e0ad237](https://github.com/stellar/js-soroban-client/commit/e0ad237d738be32b646f77ce69e9de2327382ab5) and it should work ok :)

sure it does.. but we want to update it to use the latest and greatest.. ( otherwise, we can't claim that we've tested our release, right ? )

**Comment by esteblock** [comment]
Great, will merge and publish

**Comment by tsachiherman** [comment]
Thank you!

**Comment by esteblock** [comment]
Published versions:
@soroban-react/core@3.1.0
@soroban-react/contracts@1.6.1
@soroban-react/utils@1.1.2
@soroban-react/chains@3.1.0


I'm thinking to move all to only one package to avoid much work when 1 library changes 😅

**Comment by tsachiherman** [comment]
> Published versions: @soroban-react/core@3.1.0 @soroban-react/contracts@1.6.1 @soroban-react/utils@1.1.2 @soroban-react/chains@3.1.0
> 
> I'm thinking to move all to only one package to avoid much work when 1 library changes 😅

I don't feel that I'm sufficiently knowledgeable on the usage pattern to provide a constructive feedback here.
Yes, it would definitely make the update process easier, but I can't tell what the downstream implications of that would be.

**Comment by esteblock** [comment]
> > Published versions: @soroban-react/core@3.1.0 @soroban-react/contracts@1.6.1 @soroban-react/utils@1.1.2 @soroban-react/chains@3.1.0
> > I'm thinking to move all to only one package to avoid much work when 1 library changes sweat_smile
> 
> I don't feel that I'm sufficiently knowledgeable on the usage pattern to provide a constructive feedback here. Yes, it would definitely make the update process easier, but I can't tell what the downstream implications of that would be.

Solved using lerna!
@tsachiherman

**Pull request merged**